src/SeriesList.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import AlphaLinks from './AlphaLinks';
+import Loader from './Loader';
+import Card from './Card';
+
+class SeriesList extends Component {
```

Comment:
I think we could create a List container component that could at lease be responsible for the fetching of both the Character and the Series lists.